src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -202,7 +203,7 @@ HttpStatusCode mongoSubscribeContext
     }
 
     /* Adding format to use in notifications */
-    sub.append(CSUB_FORMAT, "JSON");
+    sub.append(CSUB_FORMAT, notificationFormatToString(requestP->attrsFormat));
```

Comment:
Could you elaborate a little bit on how requestP->attrsFormat is filled, please? (Or include the links to the parts of the code where that filling takes places, both for NGSIv1 and NGSIv2 cases).